package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "sanctuary-benchmark",
+  "description": "Basis for easy benchmarking and sharing of the results",
+  "version": "0.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "make lint test"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/sanctuary-js/sanctuary-benchmark.git"
+  },
+  "files": [],
+  "dependencies": {
+    "benchmark": "2.1.x",
+    "cli-table2": "0.2.x",
+    "micromatch": "3.0.x"
+  },
+  "devDependencies": {
+    "eslint": "4.4.x",
```

Comment:
This should be `"3.19.x"` to match the [peer dependency][1].


[1]: https://github.com/sanctuary-js/sanctuary-style/blob/v0.5.0/package.json#L20